app/coffeescripts/str/TextHelper.coffee

```diff
@@ -68,7 +68,7 @@ define [
           else
             link = match
             link = "http://" + link if link[0..3] == 'www.'
-            link = encodeURI(link).replace(/'/g, '%27')
```

Comment:
I don't think that would make a difference, encode(decode(string)) should be a no-op, and if it isn't you're probably going to corrupt the user input anyway.